### PR TITLE
[TECHNICAL SUPPORT] LPS-118306 Update snapshots

### PR DESCRIPTION
**Pull request by rodrigopaulino**
There you go, @natocesarrego.

Relevant ticket:
https://issues.liferay.com/browse/LPS-118306

### CODE REVIEW

**Comment by natocesarrego** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -88,16 +90,48 @@ function FieldBase({
 		return languageValues;
 	}, [localizedValue, editingLanguageId, name]);
 	const repeatedIndex = useMemo(() => getRepeatedIndex(name), [name]);
+	const hasLabel = (label && showLabel) || required || tooltip || repeatable;
+	const hasError = displayErrors && errorMessage && !valid;
+	const fieldDetailsId = name + '_fieldDetails';
+	let fieldDetails = '';
+	let parentDivTabIndex;
+	let parentDivAriaLabelledby;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:98]: @rodrigopaulino, please sort alphabetically the variables as much as possible. Thanks.

**Comment by natocesarrego** [comment]
@rodrigopaulino, the reported failure isn't related to your pull. Can you please perform the SF mentioned by me above on this pull? This way, if the relevant suite passes, then I can forward your pull. 🙂 

Thanks.

**Comment by rodrigopaulino** [comment]
Sure thing, @natocesarrego. Do you want me to send another one or just force push the changes?

**Comment by rodrigopaulino** [comment]
I just force pushed the changes. If you want me to close this and send another pull, please let me know.

**Comment by natocesarrego** [comment]
Thanks, @rodrigopaulino. 😄 No need to send a new pull. 😉

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**